assets/Boilerplate.java

```diff
@@ -0,0 +1,62 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.klaytn.caver.Caver;
+import io.github.cdimascio.dotenv.Dotenv;
+import okhttp3.Credentials;
+import org.web3j.protocol.http.HttpService;
+
+/**
+ * Boilerplate code about "How to ..."
+ * Related article - Korean:
+ * Related article - English:
+ */
+public class Boilerplate {
+    // You can directly input values for the variables below, or you can enter values in the caver-java-examples/.env file.
+    private static String nodeApiUrl = ""; // e.g. "https://node-api.klaytnapi.com/v1/klaytn";
+    private static String accessKeyId = ""; // e.g. "KASK1LVNO498YT6KJQFUPY8S";
+    private static String secretAccessKey = ""; // e.g. "aP/reVYHXqjw3EtQrMuJP4A3/hOb69TjnBT3ePKG";
+    private static String chainId = ""; // e.g. "1001" or "8217";
+    private static String senderAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+    private static String senderPrivateKey = ""; // e.g. "0x42f6375b608c2572fadb2ed9fd78c5c456ca3aa860c43192ad910c3269727fc7"
+
+
+    public static void main(String[] args) {
+        try {
+            loadEnv();
+            run();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static String objectToString(Object value) throws JsonProcessingException {
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        return ow.writeValueAsString(value);
+    }
+
+    public static void loadEnv() {
+        Dotenv env = Dotenv.configure().directory("../../").ignoreIfMalformed().ignoreIfMissing().load();
+        if(env.get("NODE_API_URL") == null) {
+            // This handle the situation when user tries to run BoilerPlate code from project root directory
+            env = Dotenv.configure().directory(System.getProperty("user.dir")).ignoreIfMalformed().ignoreIfMissing().load();
```

Comment:
What is "user.dir" property?

Reply:
@sirano11 
It's the directory where java was run from, where you started the JVM. Does not have to be within the user's home directory. It can be anywhere where the user has permission to run java.

I added that because user can use this project using their own IDE like Intelli J.
If you open this project using Intelli J, every scenarios identified as submodules and you can run this scenarios by clicking "Run" button on Intelli J.
At this time, `user.dir` is determined as the project root directory `/path/to/caver-java-examples`, so scenarios can detect `caver-java-examples/.env` file.